tests/conftest.py

```diff
@@ -118,3 +149,12 @@ def koji_tag():
         'id_': '15638',
         'name': 'rhos-13.0-rhel-7-pending'
     })[0]
+
+
+@pytest.fixture
+def modulebuild_getTag():
```

Comment:
Optional: This should really be `module_build_getTag` to match naming standards.